ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -305,6 +310,7 @@ public long getSize() {
       if (operationException.get() != null) {
         completeRead();
       }
+      readyForPollCallback.onPollReady();
```

Comment:
Yes - it could be that the initial chunks were fetched but was not getting streamed out because the `asyncWritableChannel` was not available.